test/fixtures/kafka.go

```diff
@@ -25,10 +25,10 @@ import (
 	"time"
 )
 
-func CreateKafkaTopic() string {
+func CreateKafkaTopic(partitions, replicas string) string {
```

Comment:
Expect to pass in integers instead of strings. Let the function parse it and pass to the underlying API, preventing user input error.

```suggestion
func CreateKafkaTopic(partitions, replicas int32) string {
```